src/modules/General/components/Slider/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { Slide } from '@mui/material';
+import React, { useRef } from 'react';
+
+import css from './index.module.scss';
+import { SliderProps } from './index.types';
+import CloseButton from '../CloseButton';
+
+export const Slider: React.FC<SliderProps> = ({ open, onClose, children, title }) => {
+  return (
+    <Slide direction="left" in={open} mountOnEnter unmountOnExit>
```

Comment:
can you generalize more this component?
all props should come from the parent and better to have the default value
`direction` and {...props} for `Slide`